eng/pipelines/templates/steps/workload-publish.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+- name: publishToFeed
+  displayName: Publish to feed
+  type: boolean
+  default: false
+- name: feedForPublishing
+  displayName: Feed for publishing
+  type: string
+  default: public/dotnet10-workloads
```

Comment:
So, this parameter format is generally only used for pipeline parameters, not template parameters, as `displayName` doesn't apply here. If you look at some of the other template files, you'll see the simpler format used of `name: value`.

Reply:
Yes, that looks a lot better, thanks!